### ISSUE TITLE
Algo can't take 'sgd'

### ISSUE THREAD

**Issue by DannyWeitekamp** [vlimant/mpi_learn#1]
Try the code Algo("adadelta",loss="binary_crossentropy")
The arguments don't propagate properly.

Cheers

**Comment by DannyWeitekamp** [comment]
woops should be 
Algo("sgd",loss="binary_crossentropy")

**Comment by duanders** [comment]
The SGD learning rate has no default value, so you have to provide one:

`Algo("sgd",loss="binary_crossentropy", learning_rate=1.0)`

should work.  I agree it's confusing though.  The default learning rate is 0.01 in keras so we could implement that as the default here.

**Comment by duanders** [comment]
Fixed by https://github.com/duanders/mpi_learn/commit/0b55903e8e6adedb629cac5c5dc2f33e9d38063b.
